specification/cognitiveservices/data-plane/AnomalyDetector/stable/2022-08-23/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,987 @@
+{
+  "swagger": "2.0",
+  "info": {
+      "title": "Multivariate Anomaly Detection API",
+      "version": "v1.1",
```

Comment:
Shall we change the value in version from v1.1 to 1.1? I see FM and CV are using something like this
![image](https://user-images.githubusercontent.com/81400625/186344035-1d3ff1ed-5db0-4a26-a06f-95e3db66544c.png)

Reply:
fixed.